06_source/ADAS/ADAS_NavU/ADAS_Cfg.h

```diff
@@ -15,21 +15,16 @@
 #define ADAS_DEBUG2
 
 //PLS Buffer
-/** Size of PLS receiving buffer in bits */
+/** Size of PLS receiving buffer in bytes */
 #define PLS_RCV_BUFF_SIZE                                   500U
-/** Size of PLS sending buffer in bits */
+/** Size of PLS sending buffer in bytes */
 #define PLS_SND_BUFF_SIZE                                   50U
 	
 //ICC Buffer
 /** Size of ICC receiving buffer in bits */
```

Comment:
For the ICC buffer it is as well bytes instead of bits. My fault. 
```suggestion
/** Size of ICC receiving buffer in bytes*/
```